Makefile

```diff
@@ -22,7 +22,11 @@ test-diff:
 
 # run integration tests
 test-integration:
```

Comment:
Might be better to make this `test-integration-short` and the other, `test-integration` to stay consistent with the pattern used by `test` and `test-short`

(would require a change in yml file too)